src/synthesizer/parametric/metal_dist.py

```diff
@@ -0,0 +1,267 @@
+        if metallicity is not None:
+            self.metallicity_ = metallicity
+            self.log10metallicity_ = np.log10(self.metallicity_)
+        else:
+            self.log10metallicity_ = log10metallicity
+            self.metallicity_ = 10**self.log10metallicity_
+
+    def get_metallicity(self, *args):
+        """
+        Return the single metallicity.
+
+        NOTE: DeltaConstant is a special case where get_metallicity is overrode
+        instead of _metallicity because a single metallicity must always be
+        returned for no inputs.
+        """
+
+        # Check for bad behaviour
+        if len(args) > 0:
+            raise exceptions.InconsistentArguments(
+                "A DeltaConstant metallicity distribution takes no arguments."
+                "It can only return a single value and should not be passed"
+                "any arguments."
+            )
+
+        return self.metallicity_
+
+    def get_log10_metallicity(self, *args):
+        """
+        Return the log base 10 of the single metallicity.
+
+        NOTE: DeltaConstant is a special case where get_log10_metallicity is
+        overrode instead of _log10_metallicity because a single metallicity
+        must always be returned for no inputs.
+        """
+
+        # Check for bad behaviour
+        if len(args) > 0:
+            raise exceptions.InconsistentArguments(
+                "A DeltaConstant metallicity distribution takes no arguments."
+                "It can only return a single value and should not be passed"
+                "any arguments."
+            )
+
+        return self.log10metallicity_
+
+
+class Normal(Common):
+    """
+    A normally distributed metallicity distribution.
+
+    Attributes:
+        mean (float)
+            The mean of the normal distribution.
+        sigma (float)
+            The standard deviation of the normal distribution.
+    """
+
+    def __init__(self, mean, sigma):
+        """
+        Initialise the metallicity distribution and parent.
+
+        Either metallicity or log10metallicity must be provided.
+
+        Args:
+            mean (float)
+                The mean of the normal distribution.
+            sigma (float)
+                The standard deviation of the normal distribution.
+        """
+
+        # Instantiate the parent
+        Common.__init__(
+            self,
+            name="Normal",
+            mean=mean,
+            sigma=sigma,
+        )
+
+        # Define this models parameters
+        self.mean = mean
+        self.sigma = sigma
+
+    def _metallicity(self, metal):
+        """
+        Return the dsitrbution with at a metallicity.
+
+        Args:
+            metal (float)
+                The (linear) metallicity at which to evaluate the distribution.
+        """
+        norm = 1 / (self.sigma * np.sqrt(2 * np.pi))
+        exponent = ((metal - self.mean) / self.sigma) ** 2
+        return norm * np.exp(-0.5 * exponent)
```

Comment:
This is the same as the DeltaConstant parameterisation, right? Returns a single value?

Reply:
It returns a single value yes but unlike `DeltaConstant` it returns the weight of the distribution at `metal`. `DeltaConstant` is a weird special case that behaves differently. I wasn't going to bring it up until a later PR but it's actually pointless since the same behaviour is achieved much more simply via the `instant_metallicity` argument to `paramertic.Stars`.

You have made me realise I'd made a mistake in the methods though. The log variant of the methods is only applicable to `DeltaConstant`. I've fixed this on the most recent commit and fixed the above doc string.

Reply:
Ah, got it.

Reply:
Also just committed a general improvement to the nomenclature in these methods to make this distinction clearer.